tap_workday_raas/discover.py

```diff
@@ -8,39 +8,94 @@
 
 LOGGER = singer.get_logger()
 
-def _type_to_schema(elem_type):
+def _element_to_schema(element):
+    elem_type = element.attrib['type'].split(':')[1]
+    is_nullable = element.attrib.get('minOccurs') == '0'
+    # NB: The only actual value I've seen for maxOccurs is "unbounded". There may be other possible values
```

Comment:
In this case, it should probably be defensive and fail if it sees a value other than unbounded, that way we can account for it if it comes up.

Reply:
Done.